Helper/Order.php

```diff
@@ -1248,7 +1273,7 @@ private function setOrderState($order, $state)
         if ($state == OrderModel::STATE_HOLDED) {
             // Ensure order is in one of the "can hold" states [STATE_NEW | STATE_PROCESSING]
             // to avoid no state on admin order unhold
-            if ($prevState != OrderModel::STATE_NEW) {
+            if ($prevState != OrderModel::STATE_PROCESSING) {
```

Comment:
Should we change `!=` to `!==`?
As you did it above.

Reply:
@YevhenShBolt done